bin/release_android

```diff
@@ -17,22 +17,41 @@ if [ -z "$VERSION" ]; then
     exit 1
 fi
 
-USER=$2
-if [ -z "$USER" ]; then
-    print_error "Missing Bintray API user"
-    exit 1
-fi
+URL="https://oss.sonatype.org/service/local/staging/deploy/maven2"
```

Comment:
Looks like `SONATYPE_USERNAME` and `SONATYPE_PASSWORD` variables are missing

Reply:
Added it.